micro-ui/web/micro-ui-internals/packages/modules/workbench/src/pages/employee/SidebarConfig.js

```
@@ -0,0 +1,49 @@
+        );
+    }
+    return (
+        <div>
+            <InboxSearchComposer
+                configs={SidebarDataSearchConfig?.[0]}
+                additionalConfig={{
+                    resultsTable: {
+                        onClickRow,
+                    },
+                }}
+            ></InboxSearchComposer>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)